<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QMainWindow" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1954</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(110, 204, 175);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1934</width>
         <height>815</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1920</width>
         <height>815</height>
        </size>
       </property>
       <widget class="QFrame" name="login_style_frame_3">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>60</y>
          <width>1361</width>
          <height>161</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="Password_lineEdit">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>550</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 249, 249);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="username_label">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>490</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QFrame" name="login_style_frame_2">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>409</y>
          <width>441</width>
          <height>201</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 231, 146);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="login_pushButton">
        <property name="geometry">
         <rect>
          <x>1050</x>
          <y>600</y>
          <width>93</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 249, 249);</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="username_lineEdit">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>500</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 249, 249);</string>
        </property>
       </widget>
       <widget class="QLabel" name="invalid_login_label">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>680</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="logo_label">
        <property name="geometry">
         <rect>
          <x>381</x>
          <y>90</y>
          <width>1301</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SimSun</family>
          <pointsize>48</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#ffffff;&quot;&gt;  Forsyth Family Practice Center&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="password_label">
        <property name="geometry">
         <rect>
          <x>850</x>
          <y>540</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="logo_image_label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>0</y>
          <width>261</width>
          <height>281</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Images/Images/steth.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QFrame" name="login_style_frame">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>450</y>
          <width>451</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <zorder>login_style_frame_2</zorder>
       <zorder>login_style_frame</zorder>
       <zorder>login_style_frame_3</zorder>
       <zorder>username_label</zorder>
       <zorder>login_pushButton</zorder>
       <zorder>username_lineEdit</zorder>
       <zorder>invalid_login_label</zorder>
       <zorder>logo_label</zorder>
       <zorder>password_label</zorder>
       <zorder>logo_image_label</zorder>
       <zorder>Password_lineEdit</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1954</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
